staff/robert-anthony/precurso/master/styles/styles.css

```diff
@@ -0,0 +1,229 @@
+/*
+font-family: 'Faster One', cursive;
+font-family: 'Codystar', cursive;
+font-family: 'Sarina', cursive;
+font-family: 'Monofett', cursive;
+font-family: 'Henny Penny', cursive;
+font-family: 'Nova Mono', monospace;
+font-family: 'Bad Script', cursive;
+font-family: 'Fira Mono', monospace;
+
+*/
+.textSnazz, #scoreBoard #cerrar {
+  background: rgba(0, 255, 21, 0.58);
+  border: solid 1px rgba(0, 255, 100, 0.6); }
+
+.textSnazz {
+  display: inline-block;
+  margin-right: 30%;
+  padding: .5em; }
+
+.letterBig {
+  background-color: rgba(78, 91, 255, 0.8);
+  transform: scale(2);
+  color: red;
+  z-index: 20;
+  border: double 1px rgba(0, 200, 255, 0.8); }
+
+.letterCorrect {
+  border: double 1px rgba(0, 255, 100, 0.8);
+  background-color: rgba(1, 255, 155, 0.5); }
+
+.letterIncorrect {
+  border: double 1px rgba(255, 100, 0, 0.8);
+  background-color: rgba(255, 0, 0, 0.7); }
+
+.letterPasapalabra {
+  border: double 1px rgba(255, 255, 100, 0.7);
+  background-color: rgba(255, 255, 0, 0.7); }
+
+.letterNormal {
+  background-color: rgba(151, 10, 150, 0.85);
+  color: white;
+  border: double 1px rgba(200, 0, 200, 0.8); }
+
+* {
+  box-sizing: border-box; }
+
+html {
+  height: 100%; }
+
+body {
+  color: white;
+  background: url("../images/ghibliroad.jpg") no-repeat;
+  background-attachment: fixed;
+  background-position: left bottom -20px;
+  background-size: cover;
+  -webkit-background-clip: padding-box;
+  background-clip: padding-box;
+  background-color: skyblue;
+  font-family: "Open Sans", sans-serif;
+  border: 12px solid rgba(27, 129, 46, 0.87);
+  padding: 0;
+  margin: 0;
+  display: grid;
+  height: 100%;
+  grid-template-rows: 70px 1fr;
+  grid-template-columns: 640px 1fr;
+  grid-gap: 5px;
+  grid-template-areas: "pageHeader pageHeader " "letterDisplay userInteraction "; }
+
+#letter-display {
```

Comment:
here you are using chain-case, apply uniformly the same style guide in all your code

Reply:
👍🏻